Helm test suites failing in master

<!-- **Are you in the right place?**
1. For issues or feature requests, please create an issue in this repository.
2. For general technical and non-technical questions, we are happy to help you on our [Rook.io Slack](https://slack.rook.io/).
3. Did you already search the existing open issues for anything similar? -->

**Is this a bug report or feature request?**
* Bug Report

**Deviation from expected behavior:**
The helm suites are failing in master since #8182 merged. As seen in the [first build after the merge](https://github.com/rook/rook/actions/runs/970549954):

```
Run tests/scripts/minikube.sh helm
+ tests/scripts/minikube.sh helm
 copying rook image for helm
X Exiting due to ENV_DRIVER_CONFLICT: 'none' driver does not support 'minikube docker-env' command
Error: Process completed with exit code 1.
```

@subhamkrai Did we miss something in that merge? Strange that the CI was green before merge and now failing.

**Expected behavior:**
The helm suites to pass

--- a/.github/workflows/integration-tests-on-release.yaml
+++ b/.github/workflows/integration-tests-on-release.yaml
@@ -80,9 +80,9 @@ jobs:
         go-version: 1.16
 
     - name: setup minikube
-      uses: manusa/actions-setup-minikube@v2.4.2
+      uses: manusa/actions-setup-minikube@v2.3.1
       with:
-        minikube version: 'v1.21.0'
+        minikube version: 'v1.18.1'
         kubernetes version: ${{ matrix.kubernetes-versions }}
         start args: --memory 6g --cpus=2
         github token: ${{ secrets.GITHUB_TOKEN }}

